input/taproot_test.go

```diff
@@ -1135,7 +1136,8 @@ func TestTaprootCommitScriptToSelf(t *testing.T) {
 		},
 	}
 
-	for i, testCase := range testCases {
+	for i, testCase := range testCases { //nolint:paralleltest
```

Comment:
IMO we can also disable that linter in general as there are good reasons for not running some tests in parallel some times.